From e077c012eacfa8a6e72590d8f844e25a732bf1a1 Mon Sep 17 00:00:00 2001
From: Xilin Wu <sophon@radxa.com>
Date: Thu, 9 Oct 2025 18:29:16 +0800
Subject: [PATCH 023/268] clk: qcom: gcc-sc7280: Allow PCIe_0 GDSC to enter
 retention state

This allows suspend and resume to work properly on this platform.

Signed-off-by: Xilin Wu <sophon@radxa.com>
---
 drivers/clk/qcom/gcc-sc7280.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/clk/qcom/gcc-sc7280.c
+++ b/drivers/clk/qcom/gcc-sc7280.c
@@ -3101,7 +3101,7 @@ static struct gdsc gcc_pcie_0_gdsc = {
 	.pd = {
 		.name = "gcc_pcie_0_gdsc",
 	},
-	.pwrsts = PWRSTS_OFF_ON,
+	.pwrsts = PWRSTS_RET_ON,
 	.flags = VOTABLE,
 };
 
